README.md

```diff
@@ -1,21 +1,46 @@
 # Destiny - Cross-platform Magic Wormhole graphical client
 
-⚠ This project is still a work in progress, use at your own risk ⚠
+## An end-to-end encrypted app to share files without revealing your identity 
 
-Easily send and receive files without signup or loss of privacy. Based on the
-[wormhole-william](https://github.com/LeastAuthority/wormhole-william)
+Destiny is a secure file transfer application that allows people to transfer files without needing to reveal their identities to each other or the service provider. All files are end-to-end encrypted, meaning no one except the sender and the receiver can decrypt the contents. Users select a file on their device and then share the generated code with the intended recipient for safe delivery. No sign-up is needed.
```

Comment:
(Hmm, I was hoping the "edit" API I clicked on would "propose a change" but instead I think it just inserted it into the PR). I re-worded an awkward sentence in here and moved link-text lower down.